src/services/severa-api-service.ts

```diff
@@ -213,6 +247,38 @@ export const CreateSeveraApiService = (): SeveraApiService => {
 
       const url = `${baseUrl}/v1/workhours?eventDateStart=${startDate}&eventDateEnd=${endDate}`;
 
+      const response = await fetch(url, {
+        method: "GET",
+        headers: {
+          Authorization: `Bearer ${await getSeveraAccessToken()}`,
+          "Client_Id": process.env.SEVERA_DEMO_CLIENT_ID,
+          "Content-Type": "application/json",
+        },
+      });
+
+      if (!response.ok) {
+        throw new Error(
+          `Failed to fetch work hours: ${response.status} - ${response.statusText}`,
+        );
+      }
+      return response.json();
+    },
+
+    /**
+     * Gets previous workweeks Workhours from Severa
+     *
+     * @param severaUserId Severa user id
+     * @returns Previous workweek workhours for the user
+     */
+    getPreviousWeekHours: async (severaUserId: string) => {
+      const demoDataDate = process.env.DEMO_DATA_DATE;
+      const today = demoDataDate ? DateTime.fromISO(demoDataDate).toISODate() : DateTime.now().toISODate();
```

Comment:
This is good, lets do the same for the getPreviousWorkHours function to replace the isProduction env.
I think it should be fine to use the same env, although if that env isn't set as 2024-11-26 then I guess we can expect the data to change. But that's not an issue as we can reconfigure the demo data today together.